packages/wallets/src/utils/generate-keys.ts

```diff
@@ -1,9 +1,6 @@
-import { sha256 as SHA256 } from 'sha.js';
+import crc32 from 'crc-32';
 
 export default function generateKey(inputString: string) {
-    // Convert the string to an array buffer
-    const key = new SHA256().update(inputString).digest('hex');
-
-    // NOTE: If there are key collisions, increase the substring length
-    return key.substring(0, 8);
+    // CRC32 to generate translation keys which generates keys faster than SHA256 and detect string changes
+    return crc32.str(inputString);
```

Comment:
This generates a key based on cyclic redundancy check, which is more suitable to detect changes in raw data and has less hash key length (2^8 ~ 32 bits) compared to SHA256 (2^256 bits)